modules/hub/pages/hub-api-usage.adoc

```diff
@@ -0,0 +1,7 @@
+[[hub-api-namespaces]]
+= Hub XMRPC API Usage
+
+hub-xmlrpc-api is a daemon, currently to be launched from the command line.
+
+Once running, you can connect to the hub-xmlrpc-api at port 8888 via any XMLRPC compliant client libraries.
```

Comment:
Please always use the proper name "Hub XMLRPC API" when referring to the component in general and use backticks (`hub-xmlrpc-api`) only when referring to the executable file (or the systemd unit, when it is implemented).